SECURITY.md

```diff
@@ -0,0 +1,60 @@
+# Security Process and Policy
+
+This document provides the details on the Veriason security policy and details the processes
+surrounding security handling including a how to guide on reporting a security vulnerability
+for anything within the Veraison organization.
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users who report vulnerabilities
+to the Veraison community. All reports are thoroughly investigated by a set of Veraison maintainers.
+
+To make a report please email the private security list at ___@___ with the details.
+
+### When To Send A Report
+
+You think you have found a vulnerability in a Veraison project or a dependency of a Veraison project. This can be any of the repositories on the [Veraison GitHub organization](https://github.com/veraison).
+
+### Security Vulnerability Response
+
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
```

Comment:
We probably want to drop the strict time bound here.  The important thing to signal is we are committed to respond as swiftly as possible:

```suggestion
Each report will be reviewed and receipt acknowledged in a timely manner. This will set off the security review process detailed below.
```